modules/go-compatibility-and-support-matrix.adoc

```diff
@@ -12,15 +12,17 @@ In the table, features are marked with the following statuses:
 * *GA*: _General Availability_
 
 |===
-|*OpenShift GitOps* 6+|*Component Versions*|*OpenShift Versions*|*Support status*
```

Comment:
Please add RH SSO to the matrix.  https://spaces.redhat.com/display/GITOPS/GitOps+Component+Matrix

Reply:
Updated. Thanks.